avocado/utils/sysinfo.py

```
@@ -394,3 +406,57 @@ def collect(self):
+        """
+        Determine if 'sudo' should be used for a specific command based on the configuration.
+
+        Args:
+            cmd (str): The command to check.
+
+        Returns:
+            bool: True if 'sudo' should be used, False otherwise.
+        """
+        try:
+            first = shlex.split(cmd or "")[0]
+        except Exception:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)